apps/web/components/QuickCheckout/QuickCheckout.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <UiModal
+    @mousemove="endTimer()"
+    v-if="isOpen"
+    v-model="isOpen"
+    tag="section"
+    class="h-full md:h-fit m-0 p-0 lg:w-[1000px] overflow-y-auto"
+    aria-labelledby="quick-checkout-modal"
+  >
+    <header>
+      <h2 class="font-bold font-headings text-lg leading-6 md:text-2xl">
+        <span>{{ t('quickCheckout.heading') }}</span>
+      </h2>
+      <div class="absolute right-2 top-2 flex items-center">
+        <span v-if="timer && config.public.enableQuickCheckoutTimer" class="mr-2 text-gray-400">{{ timer }}s</span>
```

Comment:
`config.public.enableQuickCheckoutTimer` can we have this info in the composable?